rtgui/controlspotpanel.cc

```diff
@@ -660,7 +660,7 @@ void ControlSpotPanel::on_button_delete()
     nbSpotChanged_ = true;
     selSpotChanged_ = true;
     eventType = SpotDeletion;
-    SpotRow* const delSpotRow = getSpot(selIndex);
+    std::unique_ptr<SpotRow> const delSpotRow = getSpot(selIndex);
```

Comment:
@Lawrence37 This is a nice cleanup. Would you mind to change `std::unique_ptr<SpotRow> const` to `const std::unique_ptr<SpotRow>` to align it with the rest of the code?

Reply:
Updated